aiden_app/tools/wtj_scraper_tool.py

```diff
@@ -0,0 +1,14 @@
+from .tool import Tool
+from .utils.wtj_scraper import WelcomeToTheJungleScraper
+
+
+class WelcomeToTheJungleScraperTool(Tool):
+    def __init__(self):
+        self.scraper = WelcomeToTheJungleScraper()
+        super().__init__("IndeedScraper")
+        self.add_tool("search_jobs", self.search_jobs_wrapper)
+
+    @Tool.tool_function
+    def search_jobs_wrapper(self, search_query: str, location: str, num_results: int = 15, start: int = 0):
+        results = self.scraper.search_jobs(search_query, location, int(num_results), int(start))
+        return results
```

Comment:
this is not satisfying nor ready to be merged, the tool will simply return its own data format which has nothing to do with indeed's
furthermore, we need to de duplicate offers maybe